docs/hub/collections.md

```diff
@@ -0,0 +1,62 @@
+# Collections
+
+Use Collections to group repositories from the Hub (Models, Datasets, Spaces and Papers) on a dedicated page.
+
+Collections have many use cases:
+
+- Highlight specific repositories on your personal or organization profile.
+- Separate key repositories from others for your profile visitors.
+- Showcase and share a complete project with its paper(s), dataset(s), model(s) and Space(s).
+- Bookmark things you find on the Hub in categories.
+- Have a dedicated page of relevant things to share with others.
+
+This is just a list of possible uses, but remember that collections are just a way of grouping things together, so use them however you like.
+
+## Creating a new collection
+
+There are several ways to create a collection:
+
+- For personal collections: Use the **+ New** button on your logged-in homepage (1).
+- For organization collections: Use the **+ New** button available on organizations page (2).
+
+It's also possible to create a collection on the fly at the moment adding the first item from a repository page, select **+ Create new collection** from the dropdown menu (3).
+You'll need to enter a title and short description for your collection to be created.
+
+## Adding items to a collection
+
+There are 2 ways to add items to a collection.
+
+- From any repository page: Use the context menu available on any repository page then select **Add to collection** to add it to a collection (1).
+- From the collection page: If you know the name of the repository you want to add, use the **+ add to collection** option in the right-hand menu (2).
+
+It's possible to add external repositories to your collections, not just your own.
+
+## Collaborating on collections
+
+organization collections are a great way to build collections together. Any member of the organization can add, edit and remove items from the collection.
+Use the **history feature** to keep track of who has edited the collection.
+
+## Collection options
+
+### Collection visibility
+
+**Public collections appear at the top of your profile or organization page and can be viewed by anyone. The first 3 items in each collection are visible directly in the collection preview, to see more the user must click to go to the collection page.
+Set your collection to **private\*\* if you don't want it to be accessible via its URL (it will not be displayed on your profile/organization page). For organizations, private collections are only available to members of the organization.
```

Comment:
```suggestion
Set your collection to **private** if you don't want it to be accessible via its URL (it will not be displayed on your profile/organization page). For organizations, private collections are only available to members of the organization.
```